README.md

```diff
@@ -420,7 +443,6 @@ server:
     # Certificates are automatically issued and renewed if this section
     # is present.
     # There is no need in cert_file and key_file if this section is present.
-    # Autocert requires application to listen on :80 port for certificate generation
```

Comment:
Why this line has been removed?

Reply:
It was replaced with config/testdata/full.yml where this line was absent. Will update add it in both places